mountpoint-s3-client/src/s3_crt_client.rs

```diff
@@ -458,12 +459,12 @@ impl S3CrtClientInner {
                 let log_level = status_code_to_log_level(http_status);
```

Comment:
Does this make sense for "CRT requests"? Maybe we should log all of them at DEBUG, regardless of the status. Failures at the meta-request level will be logged at WARN anyway in `on_finish`.

Reply:
At the moment this also logs `http_status==-1` at WARN.